.github/workflows/cli.yml

```diff
@@ -2,10 +2,10 @@ name: cli
 on:
   push:
     branches:
-    - master
+    - api-refactor
```

Comment:
What's this branch used for

Reply:
Just for CI, need to roll back before merge.

Reply:
We can roll back because the CI passes.